src/HUDL/HUDL.cpp

```diff
@@ -31,35 +31,58 @@ CO_OBJ_T* HUDL::getObjectDictionary() {
     return &objectDictionary[0];
 }
 
-uint16_t HUDL::getObjectDictionarySize() const {
+uint16_t HUDL::getObjectDictionarySize() {
     return OBJECT_DICTIONARY_SIZE;
 }
 
 void HUDL::updateLCD() {
+    //    lcd.writeText("Hello World", 1, 0, false);
     char tempOne[32];
-    std::sprintf(tempOne, "%lu C", *(this->thermTemps + 0));
+    std::sprintf(tempOne, "%d C", *(this->thermTemps + 0));
```

Comment:
```suggestion
    std::sprintf(tempOne, "%d C", thermTemps[0]);
```
This would look a little cleaner